doc/introduction/circuits.rst

```diff
@@ -251,97 +251,6 @@ For example:
     result = circuit(0.543)
 
 
-Collections of QNodes
----------------------
```

Comment:
What is actually the preferred way to do this here?

Reply:
I think `qml.batch_params` is a work in progress.  @josh146 ?

While there's not necessarily a perfect alternative right now, this isn't going to be the way to do things going forward.  QNode collections still exist, we just don't have to tell people about them on the first page of introductory material.

Reply:
Yep, that's upcoming. However, that will allow the batching of parameters, whereas QNodeCollections allow using the same quantum function to define multiple QNodes where the observables are different.

For statevector simulators, we may be able to use measurements on the same wires, but I'm not so sure this would work with HW simulators (e.g., the QVM).

Overall, I agree that these features have not been added recently, yet, I think removing this section completely would cease to showcase a feature that can be valuable for users.

Reply:
I'm ambivalent.  If others think we should keep it, I'm fine with that.

Reply:
I have no main feeling either way 🙂